docs/proposals/20201020-management-cluster-operator.md

```diff
@@ -532,7 +532,7 @@ spec:
 apiVersion: management.cluster.x-k8s.io/v1alpha1
 kind: InfrastructureProvider
 metadata:
- name: azure
+ name: myazure
```

Comment:
I will live azure here, I don't think we should encourage to use custom names even if in this case the fetch config is specified

Reply:
Hmm...I wanted to show an example where if a different name other than the default expected one is provided, then the user needs to specify a fetch configuration.